src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -56,12 +59,13 @@ internal class HttpInListener : ListenerHandler
     private const string UnknownHostName = "UNKNOWN-HOST";
 
     private static readonly Func<HttpRequest, string, IEnumerable<string>> HttpRequestHeaderValuesGetter = (request, name) => request.Headers[name];
+    private static readonly PropertyFetcher<Exception> StopExceptionFetcher = new("Exception");
```

Comment:
Curious - why is this called `StopExceptionFetcher` instead of `ExceptionPropertyFetcher`?

Reply:
Agreeing on the Stop-prefix is redundant here. I've renamed as suggested.